annoyed-alligators/socl_media/apps/users/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+class Profile(models.Model):
+    user =  models.OneToOneField(User, on_delete=models.CASCADE)
+    image = models.ImageField(default="default.jpg", upload_to="profile_pics")
+    name = models.CharField(max_length=20)
```

Comment:
This can be split up into `first_name` and `last_name` and we can fetch them from the default user model.